packages/contact-center/user-state/tests/user-state/index.tsx

```diff
@@ -17,7 +20,10 @@ describe('UserState Component', () => {
     
     render(<UserState/>);
 
-    expect(useUserStateSpy).toHaveBeenCalledWith({cc: {}, idleCodes: [], agentId: 'testAgentId'});
+    expect(useUserStateSpy).toHaveBeenCalledWith({cc: {
```

Comment:
comment. Can we add a condition for number of times called too? It should only be called once..